### PR TITLE
Marathon 1.10 bump (Resource Limits Support)

### PR DESCRIPTION
**Pull request by timcharper**
## High-level description

Add support for Mesos resource limits by bumping Marathon to 1.10.x

## Corresponding DC/OS tickets (required)

  - [D2IQ-61131](https://jira.d2iq.com/browse/D2IQ-61131) - Resource limits API
  - [D2IQ-61130](https://jira.d2iq.com/browse/D2IQ-61130) - Resource limits support in Mesos

## Checklist for component/package updates:

If you are changing components or packages in DC/OS (e.g. you are bumping the sha or ref of anything underneath `packages`), then in addition to the above please also include:

  - [x] Change log from the last version integrated (this should be a link to commits for easy verification and review): [e04033c6f..6f2a6aec4](https://github.com/mesosphere/marathon/compare/e04033c6f...6f2a6aec4)

### CODE REVIEW

**Comment by jeschkies** [comment]
What about #7049?

File: packages/marathon/buildinfo.json [hunk 1]
```diff
@@ -4,8 +4,8 @@
   ],
   "single_source": {
     "kind": "url_extract",
-    "url": "https://s3.amazonaws.com/downloads.mesosphere.io/marathon/builds/1.9.124-e04033c6f/marathon-1.9.124-e04033c6f.tgz",
-    "sha1": "5b8933377a6b34b2dee2d692b0b9dc7eb33cd62d"
+    "url": "https://s3.amazonaws.com/downloads.mesosphere.io/marathon/builds/1.10.5-c6927f1fc/marathon-docs-1.10.5-c6927f1fc.tgz",
```
Comment (jeschkies) [inline packages/marathon/buildinfo.json]: That Tarball is just 164K. Let's wait for the upload by master

**Comment by timcharper** [comment]
> What about #7049?

If we land this first then we should just close #7049

File: packages/marathon/buildinfo.json [hunk 2]
```diff
@@ -4,8 +4,8 @@
   ],
   "single_source": {
     "kind": "url_extract",
-    "url": "https://s3.amazonaws.com/downloads.mesosphere.io/marathon/builds/1.9.124-e04033c6f/marathon-1.9.124-e04033c6f.tgz",
-    "sha1": "5b8933377a6b34b2dee2d692b0b9dc7eb33cd62d"
+    "url": "https://s3.amazonaws.com/downloads.mesosphere.io/marathon/builds/1.10.7-e52782ed7/marathon-1.10.7-e52782ed7.tgz",
```
Comment (jeschkies) [inline packages/marathon/buildinfo.json]: How did you get `1.10.7` when `master` is `1.10.4`?

Comment (timcharper) [reply packages/marathon/buildinfo.json]: it's a snapshot

**Comment by timcharper** [comment]
@mesosphere-mergebot label ready for review

**Comment by jeschkies** [comment]
There must be something broken here.

**Comment by jeschkies** [comment]
It fails with
```
2020-03-31 00:39:44.575422 +0000 UTC [scallop] Error: Unknown option 'revive_offers_for_new_apps'
```

**Comment by jeschkies** [comment]
> Networking tests are failing
@wavesoft  is already aware 🙂

**Comment by wavesoft** [comment]
So, I tried to manually post the app definition from the cluster UI:

```json
{"id": "/integration-tests/docker-user-user/app-named-local-16a61aecea40", "cpus": 0.1, "mem": 32, "instances": 1, "cmd": "/opt/mesosphere/bin/dcos-shell python /opt/mesosphere/active/dcos-integration-test/util/python_test_server.py 10620", "env": {"DCOS_TEST_UUID": "16a61aecea4047e6a6928f18f8c52453", "HOME": "/"}, "healthChecks": [{"protocol": "MESOS_HTTP", "path": "/ping", "gracePeriodSeconds": 5, "intervalSeconds": 10, "timeoutSeconds": 10, "maxConsecutiveFailures": 120, "port": 10620}], "container": {"type": "DOCKER", "docker": {"image": "debian:stretch-slim"}, "volumes": [{"containerPath": "/opt/mesosphere", "hostPath": "/opt/mesosphere", "mode": "RO"}], "portMappings": [{"containerPort": 10620, "protocol": "tcp", "name": "test", "labels": {"VIP_0": "/a6be743f-bb30-4568-a69f-c9dd614f35d0:10618"}}]}, "networks": [{"mode": "container", "name": "calico"}], "constraints": [["hostname", "CLUSTER", "172.16.3.173"]], "acceptedResourceRoles": ["*", "slave_public"]}
```

And I got:
```
acceptedResourceRoles: acceptedResourceRoles can only contain * and manual-integration-tests.
```

How is the `slave_public` nowadays selected? 😄

**Comment by wavesoft** [comment]
@timcharper you will have to adjust this file in the PR to adjust the app definitions in a way that launches them on `slave_public`:
https://github.com/wavesoft/dcos/blob/ic/D2IQ-61004-add-etcd-metrics/packages/dcos-integration-test/extra/test_networking.py#L69

**Comment by timcharper** [comment]
> @timcharper you will have to adjust this file in the PR to adjust the app definitions in a way that launches them on `slave_public`:
> https://github.com/wavesoft/dcos/blob/ic/D2IQ-61004-add-etcd-metrics/packages/dcos-integration-test/extra/test_networking.py#L69

Yeah; as mentioned in Slack, these actually were just not launching on slave_public in DC/OS 2.0. Marathon was sanitizing the field if invalid values were passed, and just dropping them.

**Pull request merged**